include/translated-object.php

```diff
@@ -253,10 +253,32 @@ public function get_translations( $id ) {
 			return array();
 		}
 
+		$translations = $this->get_translations_from_term( $id );
+
+		return $this->validate_translations( $translations, $id, 'display' );
+	}
+
+	/**
+	 * Returns an array of translations of an object.
+	 *
+	 * @since 3.4
+	 *
+	 * @param int $id Object ID.
+	 * @return int[] An associative array of translations with language code as key and translation ID as value.
+	 *
+	 * @phpstan-return array<non-empty-string, positive-int>
+	 */
+	public function get_translations_from_term( $id ) {
```

Comment:
Suggesting `get_raw_translations()` to avoid telling that we use taxonomy termis in a public function.

Reply:
Done in 74aa89a66103d39c2f4e27b3a7f3317c8d14fe46